content/sql/concepts/aggregate-functions/terms/count/count.md

```diff
@@ -36,7 +36,47 @@ Suppose there's an `employees` table with the following values:
 | Michael | 80000  | 14         |
 | Pam     | 41500  | 2          |
 | Jim     | 45000  | 4          |
-| Dwight  | 55000  | 8          |
+| Dwight  | 55000  | NULL       |
+
+To find the total number of rows in the `employees` table, the given query can be used:
+
+```sql
+SELECT COUNT(*)
+FROM employees;
+```
+
+The result would be:
+
+| COUNT(\*) |
+| --------- |
+| 4         |
+
+
+```sql
```

Comment:
Since the result is the same for both queries, can we group the two sql blocks together or list them one after another with some text to explain they will result in the same output?

Reply:
@yangc95 
Thank you for your feedback.  I combined the examples and showed only one result.